src/main/app/src/app/shared/material-form-builder/model/abstract-file-form-field.ts

```diff
@@ -20,6 +20,26 @@ export abstract class AbstractFileFormField extends AbstractFormControlField {
         super();
     }
 
+    isBestandstypeToegestaan(file: File): boolean {
+        const extensies = this.fileTypes.split(/\s*,\s*/).map(s => s.trim().toLowerCase());
+        return extensies.indexOf(this.getBestandsextensie(file)) > -1;
+    }
+
+    isBestandsgrootteToegestaan(file: File): boolean {
+        return file.size <= this.fileSizeMB * 1024 * 1024;
+    }
+
+    getBestandsextensie(file: File) {
+        if (file.name.indexOf('.') < 1) {
+            return '-';
+        }
+        return '.' + file.name.split('.').pop().toLowerCase();
+    }
+
+    getBestandsgrootteMB(file: File): string {
```

Comment:
MB staat voor megabyte maar hier wordt de grootte uitgerekend in mebibytes. Ik zou de methode hernoemen naar getBestandsgrootteInMebibytes  of anders delen door 1000000